docs/topics/db/managers.txt

```diff
@@ -196,11 +196,8 @@ object, even if it would otherwise be filtered out (and hence be inaccessible)
 by the default manager.
 
 If the normal plain manager class (:class:`django.db.models.Manager`) is not
-appropriate for your circumstances, you can force Django to use the same class
-as the default manager for your model by setting the ``use_for_related_fields``
-attribute on the manager class. This is documented fully below_.
-
-.. _below: manager-types_
+appropriate for your circumstances, you can tell Django which class to use by
+...
```

Comment:
Yep, didn't elaborate cause it's likely the manager docs will need restructuring. Since it's a fair amount of work, I thought I'd validate the idea first.